### ISSUE TITLE
Forced host server pvp is not defined

### ISSUE THREAD

**Issue by Dooleylabs** [SpigotMC/BungeeCord#1667]
I've configured my bungeecord for the first time rather successfully -- I mean it works wonderfully, but this line of code always appears in the console output, even though I have no forced host in my config.

[WARNING] Forced host server pvp is not defined

I checked and there's no pvp anywhere in the config files, so I erased everything except the config and jar file and started it again. Same output. Any reason as to why this line appears? I know I can ignore it, but I like to keep a clean log.

**Comment by ravkr** [comment]
can you show us your config?

**Comment by Dooleylabs** [comment]
I've discovered the config file restructured itself in the last reboot, which is odd. It added the default pvp server back to the forced hosts, which I've removed.

This is the config in it's current state:
https://gist.github.com/Dooleylabs/dea5499dda0415e5bfa7

Again, parts of it moved automatically and were restructured, but it works as intended. What should I put as a placeholder for the lack of a forced host?
